src/caselawclient/models/documents.py

```diff
@@ -328,9 +328,13 @@ def content_as_xml_bytestring(self) -> bytes:
     def content_as_xml_tree(self) -> Any:
         return etree.fromstring(self.content_as_xml_bytestring)
 
-    def content_as_html(self, version_uri: Optional[DocumentURIString] = None) -> str:
+    def content_as_html(
+        self,
+        version_uri: Optional[DocumentURIString] = None,
+        query: Optional[str] = None,
+    ) -> str:
         results = self.api_client.eval_xslt(
-            self.uri, version_uri, show_unpublished=True
+            self.uri, version_uri, show_unpublished=True, query=query
```

Comment:
Slightly concerned about this being used for denial of service since it won't be cached, but actually what our backend does is pretty irrelevant to that compared to just random query parameters that don't hit anything.

Reply:
I *think* my response to the previous question also covers this? let's test properly in staging and keep an eye out.